networking/configuring-routing.adoc

```diff
@@ -0,0 +1,23 @@
+// Assembly filename: configuring-routing.adoc
+// Explains Router configuration.
+[id='configuring-routing’]
```

Comment:
I think there's a backtick here, too. Will you try this? [id='configuring-routing']